### ISSUE TITLE
so have you tested on large amounts of  reality images?

### ISSUE THREAD

**Issue by xiaohaipeng** [jacobhallberg/pytorch_stylegan_encoder#1]
can the code work robusted on most face images?what is the difference between your work and https://github.com/pbaylies/stylegan-encoder

**Comment by jacobhallberg** [comment]
Hi, each face may require a bit of tuning. Changing which layer to grab features from in the VGG model can make a big difference. Additionally, specialized losses help. This is particularly shown in https://github.com/pbaylies/stylegan-encoder/blob/master/encoder/perceptual_model.py 

As for the difference between this repo and the aforementioned repo, this is simply a Pytorch implementation of the same work. I just wanted to create a simplified version that is easily understandable in terms of code and accomplished similar results because I could not easily find a Pytorch implementation. 

In fact, https://github.com/pbaylies/stylegan-encoder is even better if you don't mind using Tensorflow because of more loss functions and some optimizations made in that repo.
